vochain/indexer/archive.go

```diff
@@ -85,13 +101,16 @@ func (idx *Indexer) ImportArchive(archive []*ArchiveProcess) ([]*ArchiveProcess,
 			VoteOpts:          indexertypes.EncodeProtoJSON(p.ProcessInfo.VoteOpts),
 			PrivateKeys:       indexertypes.EncodeJSON(p.ProcessInfo.PrivateKeys),
 			PublicKeys:        indexertypes.EncodeJSON(p.ProcessInfo.PublicKeys),
-			CreationTime:      creationTime,
+			CreationTime:      time.Now(),
 			SourceBlockHeight: int64(p.ProcessInfo.SourceBlockHeight),
 			SourceNetworkID:   int64(models.SourceNetworkId_value[p.ProcessInfo.SourceNetworkId]),
 			Metadata:          p.ProcessInfo.Metadata,
 			ResultsVotes:      indexertypes.EncodeJSON(p.Results.Votes),
 			VoteCount:         int64(p.ProcessInfo.VoteCount),
+			ChainID:           p.ChainID,
+			FromArchive:       true,
 		}
+
 		if _, err := queries.CreateProcess(context.TODO(), procParams); err != nil {
 			return nil, fmt.Errorf("create archive process: %w", err)
 		}
```

Comment:
you could seize the opportunity to fix the typo in the next func name `StartArchiveRetrival` -> `StartArchiveRetrieval`